src/itemTracker/Images.ts

```diff
@@ -479,6 +484,34 @@ const images: Images = {
     'Small Key': [
         cavesKey,
     ],
+    'Skyview Small Key': [
+        noSmallKey,
+        oneSmallKey,
```

Comment:
Might as well do all these properly, and try using the "first" level of each item